### PR TITLE
fix: show last deleted message [WPB-1899]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1899" title="WPB-1899" target="_blank">BugWPB-1899</a>  Wrong conversation second line when someone deletes the last message
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

All deleted messages were filtered from message previews

### Causes (Optional)

Conversations showed last message before deleted message

### Solutions

Remove deleted message previews filtering and introduce new Deleted content

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 
last_deleted_message

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [4708](https://10.10.124.134/job/AR-build-pipeline/4708/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt [hunk 1]
```diff
@@ -284,6 +286,33 @@ fun PreviewGroupConversationItemWithNoBadges() {
...
+fun PreviewGroupConversationItemWithLastDeletedMessage() {
+    ConversationItemFactory(
+        conversation = ConversationItem.GroupConversation(
+            "groupName looooooooooooooooooooooooooooooooooooong",
+            conversationId = QualifiedID("value", "domain"),
+            mutedStatus = MutedConversationStatus.AllAllowed,
+            lastMessageContent = UILastMessageContent.SenderWithMessage(
+                "John".toUIText(),
+                UIText.StringResource(R.string.deleted_message_text),
+                ": ${MarkdownConstants.NON_BREAKING_SPACE}"
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt]: on tests we have an extra space between `:` and `${Markdown...` is this expected? As in the normal implementation there is no space

**Review by alexandreferris — COMMENTED** [review]
Nice! Just got a small comment

**Comment by AndroidBob** [comment]
Build [4710](https://10.10.124.134/job/AR-build-pipeline/4710/) **failed**.

**Comment by AndroidBob** [comment]
Build [4721](https://10.10.124.134/job/AR-build-pipeline/4721/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.8.0(9661446)(9661446)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.8.0(9661446)(9661446)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Review by saleniuk — APPROVED** [review]

**Pull request merged**